packages/rancher-istio/charts/values.yaml

```diff
@@ -93,4 +93,10 @@ kiali:
       custom_metrics_url: "http://rancher-monitoring-prometheus.cattle-monitoring-system.svc:9090"
       url: "http://rancher-monitoring-prometheus.cattle-monitoring-system.svc:9090"
     tracing:
-      enabled: false
+      in_cluster_url: "http://tracing.istio-system.svc:16686"
+
+tracing:
+  enabled: true
```

Comment:
If one doesn't exist we need to make a UI issue to allow configuring this on/off.

Reply:
we discussed this going to false by default now